app/controllers/users_controller.rb

```diff
@@ -1,12 +1,13 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!, only: [:show]
   before_action :find_user, only: [:show]
 
```

Comment:
- [ ] In your `posts_controller.rb` kindly add to the top of your class `PostsController` the method `load_and_authorize_resource` from `CanCanCan` to manage authorizations.